public/js/services/Api.js

```diff
@@ -4,7 +4,7 @@ import moment from 'moment';
 export const httpClient = axios.create({
     // baseURL: 'https://productionmetrics.local.dev-gutools.co.uk/', // Use this url in dev. TODO: Add proper ENV management
     baseURL: 'https://productionmetrics.gutools.co.uk/',
-    timeout: 5000,
+    timeout: 10000,
```

Comment:
The internet was very slow and i was getting timeout errors. I can probably revert this now :)